app/assets/src/components/views/discovery/BaseDiscoveryView.jsx

```diff
@@ -16,17 +17,19 @@ class BaseDiscoveryView extends React.Component {
       handleRowClick,
       initialActiveColumns,
       protectedColumns,
+      rowClassName,
+      rowHeight,
     } = this.props;
 
     return (
       <Table
         columns={columns}
         data={data}
-        defaultRowHeight={68}
+        defaultRowHeight={rowHeight || 68}
```

Comment:
Do you know the origin of the 68?
Can you switch it to a prop with default value ?

Reply:
no idea where the 68 was from actually. Moved to default props 👍